packages/dbml-core/src/parse/dbml/src/lib/errors.ts

```diff
@@ -0,0 +1,28 @@
+export enum ParsingErrorCode {
```

Comment:
Why do we need this?

Reply:
It's a way to map a type of error to a code. It isn't strictly needed and I only include it if this may be desired in the future.

Reply:
What are the use cases in the future that would need this em?

Reply:
This is not very useful at current as it's still rudimentary. You can think of it's like HTTP 404 error code. AML has error codes, too :v.